CMakeLists.txt

```diff
@@ -337,6 +337,26 @@ endif (WIN32)
 
 endif (OVERRIDE_CMAKE_CXX_FLAGS)
 
+OPTION(CMAKE_ENABLE_CODE_COVERAGE "Enable code coverage" OFF)
+
+if(CMAKE_ENABLE_CODE_COVERAGE)
+
+  string(TOUPPER ${CMAKE_BUILD_TYPE} UPPERCASE_CMAKE_BUILD_TYPE)
+  if(NOT UPPERCASE_CMAKE_BUILD_TYPE STREQUAL DEBUG)
```

Comment:
I can see now that we enable the coverage builds for Unix only now. Given the fact that we consider Unix builds to be always using single-target generators, I would suggest moving the check for the build type inside of the `if(CLR_CMAKE_PLATFORM_UNIX)` below. That would fix my concern.